FWCore/Framework/src/EventProcessor.cc

```diff
@@ -488,20 +490,51 @@ namespace edm {
 
       processBlockHelper_ = std::make_shared<ProcessBlockHelper>();
 
-      // initialize the input source
-      input_ = makeInput(*parameterSet,
-                         *common,
-                         items.preg(),
-                         items.branchIDListHelper(),
-                         get_underlying_safe(processBlockHelper_),
-                         items.thinnedAssociationsHelper(),
-                         items.actReg_,
-                         items.processConfiguration(),
-                         preallocations_);
-
-      // initialize the Schedule
-      schedule_ =
-          items.initSchedule(*parameterSet, hasSubProcesses, preallocations_, &processContext_, *processBlockHelper_);
+      std::optional<ScheduleItems::MadeModules> madeModules;
+      {
```

Comment:
Minor, but this could be moved inside the following block, no?
```suggestion
      {
        std::optional<ScheduleItems::MadeModules> madeModules;
```

Reply:
done